source/Settleup.py

```diff
@@ -0,0 +1,87 @@
+import desert
+from group import Group, UserGroupView
+import pyrebase
+import requests
+import json
+import user_auth_config
+
+
+class Settleup:
+
+    def __init__(self):
+        config = {
+            "apiKey": "AIzaSyCfMEZut1bOgu9d1NHrJiZ7ruRdzfKEHbk",
+            "authDomain": "settle-up-sandbox.firebaseapp.com",
+            "databaseURL": "https://settle-up-sandbox.firebaseio.com",
+            "storageBucket": "settle-up-sandbox.appspot.com"
+        }
```

Comment:
This should be moved to a centralized location for configuration. Probably want a dataclass to store this information

Reply:
`__init__` would potentially take this configuration as a parameter